src/mut/writers.cpp

```diff
@@ -202,8 +208,13 @@ void asc(const Morphology& morphology, const std::string& filename) {
     }
 
     for (const auto& section : morphology.rootSections()) {
+        // allowed types are only the ones available in the header
```

Comment:
looks good to me, could we just add a named type here, or something slightly clearer?

Reply:
yes, ofc!